amazonka/src/Network/AWS/Auth.hs

```diff
@@ -98,10 +99,14 @@ envSecretKey = "AWS_SECRET_ACCESS_KEY"
 envSessionToken :: Text -- ^ AWS_SESSION_TOKEN
 envSessionToken = "AWS_SESSION_TOKEN"
 
--- | Credentials profile environment variable.
+-- | Default credentials profile environment variable.
 envProfile :: Text -- ^ AWS_PROFILE
 envProfile = "AWS_PROFILE"
 
+-- | Default region environment variable
+envRegion :: Text -- ^ AWS_REGION
+envRegion = "AWS_REGION"
```

Comment:
Just to add - this is an unfortunate misconception since really the only documentation for environment variables links to the CLI documentation. Most of the actual SDK documentation is buried in Java/C#/etc. class docs.